docs/pages/rmf-audio-capture_High-Level_TestSpec.md

```diff
@@ -0,0 +1,110 @@
+
+#### Test Startup Requirement - Check primary audio capture
+
+ - Ensure audio is playing in the background before starting L3 tests.
+
+#### Emulator Requirements - Check primary audio capture
+
+- Emulator to implement RMF_AudioCapture HAL that is able to deliver a known 10-second audio clip (from wav or raw PCM file) when triggered by control plane.
+
+#### Control Plane Requirements - Check primary audio capture
+
+- Control plane must be able to trigger emulator HAL to deliver a known 10-second audio clip.
+
+
+
+### Check auxiliary audio capture
+
+Applicable only on devices that support auxiliary capture.
+
+
+|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----------|--------|--|--|--------------------------|
+|Run auxiliary audio capture for 10 seconds and verify receipt of commensurate amount of audio samples. Verify that there are no more data ready callbacks issued after the RMF_AudioCapture_Stop returns |RMF_AudioCapture_Open_Type, RMF_AudioCapture_GetDefaultSettings, RMF_AudioCapture_Start, RMF_AudioCapture_Stop, RMF_AudioCapture_Close|`Y`|`NA`|`NA`|
+|Run auxiliary audio capture for 10 seconds with known source material and verify that captured audio is faithful to the source within margin of error|RMF_AudioCapture_Open_Type, RMF_AudioCapture_GetDefaultSettings, RMF_AudioCapture_Start, RMF_AudioCapture_Stop, RMF_AudioCapture_Close|`NA`|`Y`|`Y`|
+|Run auxiliary audio capture for 2 minutes and verify that a commensurate amount of audio data is delivered. Also verify that jitter low enough to avoid underruns with an application buffer that's half the FIFO size.|RMF_AudioCapture_Open, RMF_AudioCapture_GetDefaultSettings, RMF_AudioCapture_GetCurrentSettings, RMF_AudioCapture_Start, RMF_AudioCapture_Stop, RMF_AudioCapture_Close|`NA`|`Y`|`N`|
+
+#### Test Startup Requirement - Check auxiliary audio capture
+
+ - Ensure audio is playing in the background before starting L3 test. Test content must have auxiliary audio track.
+
+#### Emulator Requirements - Check auxiliary audio capture
+
+- Emulator to implement RMF_AudioCapture HAL that is able to deliver a known 10-second audio clip (from wav or raw PCM file) when triggered by control plane.
+
+#### Control Plane Requirements - Check auxiliary audio capture
+
+- Control plane must be able to trigger emulator HAL to deliver a known 10-second audio clip.
+
+
+### Check concurrent audio capture
+
+Applicable only on devices that support auxiliary capture.
+
+
+|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----------|--------|--|--|--------------------------|
+|Run auxiliary+primary audio capture for 10 seconds and verify receipt of commensurate amount of audio samples.  Verify that there are no more data ready callbacks issued after the RMF_AudioCapture_Stop returns |RMF_AudioCapture_Open_Type, RMF_AudioCapture_GetDefaultSettings, RMF_AudioCapture_Start, RMF_AudioCapture_Stop, RMF_AudioCapture_Close|`Y`|`NA`|`NA`|
```

Comment:
can you add point like read the profile to check aux support or not 
`rmfaudiocapture\features\auxsupport`